src/platform/plugins/shared/dashboard/server/api/register_routes.ts

```diff
@@ -56,6 +58,27 @@ const commonRouteConfig = {
   },
 } as const;
 
+const formatResult = (item: DashboardItem) => {
+  const {
+    id,
+    type,
+    attributes,
+    createdAt,
+    updatedAt,
+    createdBy,
+    updatedBy,
+    error,
+    managed,
+    ...rest
+  } = item;
+  return {
+    id,
+    type,
+    data: { ...attributes, ...rest },
+    meta: { createdAt, updatedAt, createdBy, updatedBy, error, managed },
+  };
+};
```

Comment:
Something I have previously missed. The saved object response includes a `version` property that needs to be put into the `meta`. Currently, this `version` is getting stuffed into the rest parameters and is overwriting the `attributes.version` property. I think my suggestion below should fix that.

```suggestion
  const {
    id,
    type,
    attributes,
    createdAt,
    updatedAt,
    createdBy,
    updatedBy,
    error,
    managed,
    version,
    ...rest
  } = item;
  return {
    id,
    type,
    data: { ...attributes, ...rest },
    meta: { createdAt, updatedAt, createdBy, updatedBy, error, managed, version },
  };
};
```